### PR TITLE
fix: remove logdna and AT from fscloud example

### PR DESCRIPTION
**Pull request by iamar7**
### Description

The module needs to be updated to no longer use LogDNA or Activity Tracker in the examples.

Issue: https://github.ibm.com/GoldenEye/issues/issues/11113

### Release required?

- [ ] No release
- [ ] Patch release (`x.x.X`)
- [x] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

**Comment by iamar7** [comment]
The `TestRunUpgradeAdvancedExample` is failing with the following error. Looking into the reason behind the failure.

Screenshot 2024-10-11 at 5 47 48 PM

File: examples/multiple_mzr_clusters/main.tf [hunk 1]
```diff
@@ -174,20 +174,13 @@ data "ibm_container_cluster_config" "cluster_config_c2" {
...
   resource_group_id              = module.resource_group.resource_group_id
   region                         = var.region
-  log_analysis_plan              = "7-day"
   cloud_monitoring_plan          = "graduated-tier"
-  activity_tracker_provision     = false
   enable_platform_logs           = false
   enable_platform_metrics        = false
-  log_analysis_instance_name     = "${var.prefix}-logdna"
   cloud_monitoring_instance_name = "${var.prefix}-sysdig"
   cloud_logs_provision           = false
```
Comment (Aashiq-J) [inline examples/multiple_mzr_clusters/main.tf:185]: This needs to be set to `true`.
and the [logs agents](https://github.com/terraform-ibm-modules/terraform-ibm-observability-agents/blob/main/variables.tf#L233) should send logs to the Cloud logs.

File: examples/fscloud/main.tf [hunk 2]
```diff
@@ -101,33 +100,19 @@ module "vpc" {
...
   resource_group_id              = module.resource_group.resource_group_id
   cloud_monitoring_instance_name = "${var.prefix}-sysdig"
   cloud_monitoring_plan          = "graduated-tier"
-  enable_platform_logs           = false
   enable_platform_metrics        = false
-  log_analysis_provision         = false
-  activity_tracker_instance_name = "${var.prefix}-at"
-  activity_tracker_plan          = "7-day"
-  activity_tracker_provision     = !local.existing_at
   cloud_logs_provision           = false
```
Comment (Aashiq-J) [inline examples/fscloud/main.tf:116]: This needs to be set to `true`.
and the [logs agents](https://github.com/terraform-ibm-modules/terraform-ibm-observability-agents/blob/main/variables.tf#L233) should send logs to the Cloud logs.

File: examples/multiple_mzr_clusters/main.tf (hunk 1 above)
Comment (ocofaigh) [reply examples/multiple_mzr_clusters/main.tf:185]: @Aashiq-J we dont need to test deploying log agents in base OCP module

File: examples/fscloud/main.tf (hunk 2 above)
Comment (ocofaigh) [reply examples/fscloud/main.tf:116]: same as https://github.com/terraform-ibm-modules/terraform-ibm-base-ocp-vpc/pull/542/files#r1800842243

**Review by ocofaigh — CHANGES_REQUESTED** [review]
I don't think you need to provision any Observability instances anymore in the fscloud example. The only place thats left is [here](https://github.com/terraform-ibm-modules/terraform-ibm-base-ocp-vpc/blob/ab7299be9f21351c0045489927d08671183c1e80/examples/fscloud/main.tf#L24), but its no longer required to explicitly pass a monitoring CRN to enable monitoring on a bucket.

**Review by ocofaigh — APPROVED** [review]

**Pull request merged**